Doc/library/tkinter.rst

```diff
@@ -67,13 +67,33 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1, sync=0, use=None)
+
+   Return a toplevel Tk widget, which is usually the main window of an
+   application.  Each instance has its own associated Tcl interpreter.
+
+   The :class:`Tk` class is typically instantiated using all default values.
+   However, the following keyword arguments are currently recognized:
+
+   *screenName*
+      Sets the DISPLAY environmental variable. (X11 only)
```

Comment:
```rst
:envvar:`DISPLAY`
```

Reply:
Nice!  Thanks.

Reply:
How about "When given (as a string), set the DISPLAY ..."

Are this and the other X option(s?) just ignored on other systems?

Reply:
In my initial research for documenting this, I couldn't find any reference to DISPLAY on Windows.  On the wiki page (https://en.wikipedia.org/wiki/Environment_variable), it specifically says that DISPLAY is X11 only.  As far as controlling the screen destination for Windows, most answers online say that Windows doesn't provide it natively, but there are third party options that allow it.

This sets the ``DISPLAY`` element of the ``env`` in ``interp`` for Tcl (https://www.tcl.tk/man/tcl8.5/TclCmd/tclvars.htm#M4).  Reading this document, I think it adds it to the Windows environment and Windows just doesn't use it.

One other note - DISPLAY has a specific format of ``hostname:displaynumber.screennumber``.  Should I mention that?